packages/ics721/src/ibc_packet_receive.rs

```diff
@@ -297,8 +323,12 @@ impl ActionAggregator {
         contract: Addr,
         receiver: Addr,
         callback_msg: Option<WasmMsg>,
+        incoming_proxy_msg: Option<WasmMsg>,
     ) -> StdResult<SubMsg<Empty>> {
         let mut m = Vec::with_capacity(2);
```

Comment:
Would increase it to 3, cause we now have, the ics721 message, callbacks msg and incoming msg (if needed)

Reply:
done